samples/client/petstore/cpp-qt/client/PFXHelpers.h

```diff
@@ -272,6 +251,37 @@ bool fromJsonValue(QMap<QString, T> &val, const QJsonValue &jval) {
     return ok;
 }
 
+template <typename T>
+class OptionalParam {
+public:
+    T m_Value;
+    bool m_isNull = false;
+    bool m_hasValue;
+public:
+    OptionalParam(){
+        m_hasValue = false;
+    }
+    OptionalParam(const T &val, bool isNull = false){
+        m_hasValue = true;
+        m_Value = val;
+        m_isNull = isNull;
+    }
+    bool hasValue() const {
+        return m_hasValue;
+    }
+    T value() const{
+        return m_Value;
+    }
+
+    QString stringValue() const {
+        if (m_isNull) {
+            return QStringLiteral("");
+        } else {
+            return toStringValue(value());
+        }
+    }
+};
+
```

Comment:
Same

Reply:
It is a generated code, I entered the reason above in the template

Reply:
indeed